tf_encrypted/session.py

```diff
@@ -118,22 +126,38 @@ def run(
             writer.close()
 
             if __TFE_TRACE__ or write_trace:
-                chrome_trace = timeline.Timeline(run_metadata.step_stats).generate_chrome_trace_format()
+                tl = timeline.Timeline(run_metadata.step_stats)
```

Comment:
`timeline`?

Reply:
I think `timeline` would conflict with the imported `timeline` module.

Reply:
Maybe `trace` instead?